packages/teams-js/src/public/visualMedia.ts

```diff
@@ -0,0 +1,308 @@
+       */
+      sourceProps: CameraProps;
+
+      /**
+       * The common additional properties for image in mobile devices
+       */
+      commonImageprops: ImageProperties;
+    }
+
+    /**
+     * CameraImageProperties is for the image taken from the camera
+     * @beta
+     */
+    export interface GalleryImageProperties extends VisualMediaProps {
+      /**
+       * The source in GalleryImageProperties should always be GalleryProps
+       */
+      sourceProps: GalleryProps;
+
+      /**
+       * The common additional properties for image in mobile devices
+       */
+      commonImageprops: ImageProperties;
+    }
+
+    /**
+     *  All properties in ImageProperties are optional and have default values in the platform
+     *  Additional properties for image in mobile devices
+     * @beta
+     */
+    export interface ImageProperties {
+      /**
+       * Optional; indicate if inking on the selected Image is allowed or not
+       * Default value is false
+       */
+      shouldAllowInkingOnImages?: boolean;
+
+      /**
+       * Optional; indicate if putting text stickers on the selected Image is allowed or not
+       * Default value is false
+       */
+      shouldAllowTextStickersOnImages?: boolean;
+
+      /**
+       * Optional; indicate if the filter mode is enabled on the selected image
+       * Default value is false
+       */
+      enableFilterOnImages?: boolean;
+
+      /**
+       * Optional; Lets the developer specify the image output formats, more than one can be specified.
+       * Default value is Image.
+       */
+      imageFormatsAllowedToBeSelected?: ImageOutputFormats[];
+    }
+
+    /**
+     * Specifies the image output formats.
+     * @beta
+     */
+    export enum ImageOutputFormats {
+      /** Outputs image.  */
+      Image = 1,
+      /** Outputs pdf. */
+      PDF = 2,
+    }
+    const errorInvalidSource = 'Invalid input source.';
+    const errorInvalidCount = 'Invalid input count: Must supply the valid image counts (limit to 10).';
+    const errorInvalidResponse = 'Invalid response: Received more images than the specified max limit in the response.';
+    /**
+     * Capture one or multiple image(s) using camera.
+     * @beta
+     * @param imageInputs - The input params to customize the image(s) to be captured
+     * @returns Promise that will resolve with {@link VisualMediaFile[]} object or reject with an error.
+     * @throws INVALID_ARGUMENTS Error if imageInputs is null or imageInputs.maxVisualMediaCount is greater than 10 or lesser than 1.
+     */
+    export async function captureImages(cameraImageInputs: CameraImageProperties): Promise<VisualMediaFile[]> {
+      ensureInitialized(runtime, FrameContexts.content, FrameContexts.task);
+      if (!isSupported()) {
+        throw errorNotSupportedOnPlatform;
+      }
+      if (
+        !cameraImageInputs ||
+        cameraImageInputs.maxVisualMediaCount > 10 ||
```

Comment:
Hmmmm I think in this place 'maxVisualMediaCount' doesn't make sense to me. Maximum number should be a fixed number and maybe it is better to be named as visualMediaCount?